trixnity-core/src/commonMain/kotlin/net/folivo/trixnity/core/model/events/m/room/MessageEventContent.kt

```diff
@@ -42,8 +42,8 @@ sealed class MessageEventContent : RoomEventContent {
 
     @Serializable
     data class UnknownMessageEventContent(
-        @SerialName("msgtype") val type: String,
-        @SerialName("body") override val body: String
+        @SerialName("msgtype") val type: String? = null,
```

Comment:
They are both requried in the Matrix Client-Server spec: https://matrix.org/docs/spec/client_server/r0.6.1#m-room-message

Reply:
Okay, but I got an exception that they are missing. I will debug more layer